include/sys/zio.h

```diff
@@ -163,63 +163,62 @@ typedef enum zio_suspend_reason {
 	ZIO_SUSPEND_MMP,
 } zio_suspend_reason_t;
 
-enum zio_flag {
+typedef uint64_t zio_flag_t;
 	/*
 	 * Flags inherited by gang, ddt, and vdev children,
 	 * and that must be equal for two zios to aggregate
 	 */
-	ZIO_FLAG_DONT_AGGREGATE	= 1U << 0,
-	ZIO_FLAG_IO_REPAIR	= 1U << 1,
-	ZIO_FLAG_SELF_HEAL	= 1U << 2,
-	ZIO_FLAG_RESILVER	= 1U << 3,
-	ZIO_FLAG_SCRUB		= 1U << 4,
-	ZIO_FLAG_SCAN_THREAD	= 1U << 5,
-	ZIO_FLAG_PHYSICAL	= 1U << 6,
+#define	ZIO_FLAG_DONT_AGGREGATE		(1ULL << 0)
+#define	ZIO_FLAG_IO_REPAIR		(1ULL << 1)
+#define	ZIO_FLAG_SELF_HEAL		(1ULL << 2)
+#define	ZIO_FLAG_RESILVER		(1ULL << 3)
+#define	ZIO_FLAG_SCRUB			(1ULL << 4)
+#define	ZIO_FLAG_SCAN_THREAD		(1ULL << 5)
+#define	ZIO_FLAG_PHYSICAL		(1ULL << 6)
```

Comment:
The extra level of indention here doesn't appear to be needed.  Removing it will also resolve the style warnings.

Reply:
It is needed for a flag under development, but I agree that including it here is premature. I will remove it.

Reply:
Done.